web/angular-wallet/src/app/main/send-burst/send-burst-form/send-burst-form.component.ts

```diff
@@ -63,7 +63,7 @@ export class SendBurstFormComponent implements OnInit {
       const dialogRef = this.openWarningDialog([this.recipient]);
       dialogRef.afterClosed().subscribe(ok => {
         if (ok) {
-          this.sendBurst(convertNumericIdToAddress(this.recipient.addressRaw));
+          this.sendBurst(this.recipient.addressRaw);
```

Comment:
Oh.. AddressRaw could be a burst address, or an Id.  What happens in case of id? Sendburst expects addressRS

Reply:
Good question - tested and it works.